tests/test.ts

```diff
@@ -167,6 +167,35 @@ writeToLog(\`\${words.join(' ')}\`);`);
   // ...
 }`);
   },
+  testIssue26Export: () => {
```

Comment:
Remove all references to GitHub. (no Issue26)
You can add a reference to this issue in your git commit message.
`testExport`